### PR TITLE
feat_: Change opt short names

### PR DESCRIPTION
**Pull request by Cuteivist**
Task: https://github.com/status-im/status-desktop/issues/14720

Status-desktop PR: https://github.com/status-im/status-desktop/pull/14766

### CODE REVIEW

**Review by stefandunca — APPROVED** [review]
👍

File: rpc/network/network.go [hunk 1]
```diff
@@ -157,6 +157,13 @@ func (nm *Manager) Init(networks []params.Network) error {
 					}
 				}
 
+				if networks[i].ShortName != currentNetworks[j].ShortName {
+					err := nm.updateShortName(networks[i].ChainID, networks[i].ShortName)
```
Comment (saledjenic) [inline rpc/network/network.go]: @Cuteivist could we make this update as a migration script? 
This is a different thing than rpc URL, this one is one-time change, migration from the old to the new labels and I see it as a perfect candidate for migration script.

**Review by OmarBasem — APPROVED** [review]

**Review by saledjenic — CHANGES_REQUESTED** [review]
@Cuteivist please change:
`opt` -> `oeth`
`arb` -> `arb1`

for both mainnet and testnet mode, don't use other abbreviations for testnet.

**Review by smohamedjavid — COMMENTED** [review]
We also need to update `chain_short_names` column in the `saved_addresses` table.

**Review by IvanBelyakoff — APPROVED** [review]

**Comment by Cuteivist** [comment]
> We also need to update `chain_short_names` column in the `saved_addresses` table.

@smohamedjavid  Thank you for pointing that out, easy to miss! I added additional migration for wallet to update those prefixes too.

File: walletdatabase/migrations/sql/1715839555_rename_chain_prefixes.up.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+UPDATE saved_addresses SET chain_short_names = REPLACE(chain_short_names, 'arb:', 'arb1:') WHERE chain_short_names LIKE '%arb:%';
```
Comment (saledjenic) [inline walletdatabase/migrations/sql/1715839555_rename_chain_prefixes.up.sql:1]: @smohamedjavid could you confirm that `REPLACE` works correctly on iOS?

**Review by saledjenic — APPROVED** [review]
@Cuteivist thanks for the changes, all looks good. You can merge it once @smohamedjavid confirms.

File: walletdatabase/migrations/sql/1715839555_rename_chain_prefixes.up.sql (hunk 2 above)
Comment (smohamedjavid) [reply walletdatabase/migrations/sql/1715839555_rename_chain_prefixes.up.sql:1]: Yep, It's all good 👍

**Review by smohamedjavid — APPROVED** [review]

**Pull request merged**